calmTheStorm/calmTheStorm/ContentView.swift

```diff
@@ -9,8 +9,16 @@ import SwiftUI
 
 struct ContentView: View {
     var body: some View {
-        Text("Hello, Calm the Storm Team!")
-            .padding()
+        
+        ZStack{
```

Comment:
Is there a reason for using a ZStack here? Is there some issue with using the colors with the Views we are familiar with?

Reply:
To make the background a different color you have to put it in a ZStack.